tests/charbot/test_gcal.py

```diff
@@ -0,0 +1,46 @@
+import random
+from unittest.mock import AsyncMock, Mock
+
+import discord
+import pytest
+import datetime
+import calendar
+from discord.ext import commands
+
+from charbot import gcal
```

Comment:
```suggestion
import datetime

import pytest

from charbot import gcal
```

Reply:
Doing this should resolve everything but pyright CI failing, i've figured out a fix to that and i'll push it separately